cmd/containerboot/main.go

```diff
@@ -113,6 +114,7 @@ func main() {
 		Socket:          defaultEnv("TS_SOCKET", "/tmp/tailscaled.sock"),
 		AuthOnce:        defaultBool("TS_AUTH_ONCE", false),
 		Root:            defaultEnv("TS_TEST_ONLY_ROOT", "/"),
+		AcceptRoutes:    defaultBool("TS_ACCEPT_ROUTES", false),
```

Comment:
lets not add new env vars here. Users can use `TS_EXTRA_ARGS` to do the same thing. We really want to move to a more declarative config and get rid of these env vars.

Reply:
ah got it - instead I just updated comments in main and added a test case of passing in accept-routes to TS_EXTRA_ARGS.